app/controllers/submit.js

```diff
@@ -268,15 +275,19 @@ export default Ember.Controller.extend(Analytics, BasicsValidations, NodeActions
     basicsTags: Ember.computed('node', function() {
         // Pending tags
         let node = this.get('node');
-        return node ? node.get('tags') : Ember.A();
+        let newTags = null;
+        if (node != null) {
+            newTags = node.get('tags').slice(0).map(fixSpecialChar);
```

Comment:
Per our discussion: can we stick to consistently transforming values in one place, eg fixing display of characters in the template instead of the controller?

Having a consistent standard keeps display logic in the display layer, and makes it easier to find/guess where a value is being changed later.

A major exception to that rule would be if this value were being passed to the tags widget- there is some special logic for the `jquery.tagsinput` widget, and in that case it would be ok to modify the value in the controller.